live-examples/html-examples/input/css/input.css

```diff
@@ -0,0 +1,52 @@
+@font-face {
+  font-family: 'Fira Sans';
+  src: local('FiraSans-Regular'),
+  url('/media/fonts/FiraSans-Regular.woff2') format('woff2');
+}
+
+legend {
+  background-color: #000;
+  color: #ffffff;
+  padding: 3px 6px;
+}
+
+input,
+label {
+    width: 43%;
+}
+
+input {
+  margin: 0.5rem 0;
+  padding: 0.5rem;
+  border-radius: 4px;
+  border: 1px solid #ddd;
+  outline: none;
```

Comment:
Nit: Code Style: indent with 4 spaces. Do not remove outline as it negatively impacts accessibility, unless there is alternate styling provider for the `active` and `focus` states.